partcad/src/partcad/user_config.py

```
@@ -20,6 +20,10 @@ class UserConfig(vyper.Vyper):
     def get_config_dir():
         return os.path.join(Path.home(), ".partcad")
 
+    @staticmethod
+    def get_cache_dir():
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `get_cache_dir`

(ANN205)